spinnaker-install-admin-guides.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: archive
+title: "Spinnaker Installation and Administration"
+permalink: /spinnaker-install-admin-guides/
+description: Add a description here.
+collection: spinnaker_install_admin_guides
+---
+
+This is a placeholder to point at the custom archive.html layout.
```

Comment:
Just wanted to make sure that you were leaving this here on purpose.

Reply:
Yep, this is intentional!